src/erlcloud_ddb2.erl

```diff
@@ -1754,6 +1772,25 @@ put_item(Table, Item, Opts) ->
 %%         [{<<":f">>, <<"Amazon DynamoDB">>},
 %%          {<<":s">>, <<"How do I update multiple items?">>}]}]),
 %% '
+%%
+%% The ConditionExpression option can be used in place of the legacy Expected parameter.
```

Comment:
This also seems to more or less match the previous example now. Please remove one.